### ISSUE TITLE
Dynamically change tooltip attachment point

### ISSUE THREAD

**Issue by kirillminiaev** [HubSpot/tooltip#53]
Is it possible to modify a tooltip's attachment point dynamically? Case-in-point: when I resize my window size, some of my tooltips fall outside the window bounds. Can tooltips do one of two things:

(1) Listen to window size and modify their horizontal position to never fall outside the window bounds
(2) Allow me to put a resize listener (jQuery) and when the window is of a specific size, change the tooltip to a different attachment point (i.e. change 'top center' to 'top right')

Thank you.

**Comment by dottodot** [comment]
This is very much needed, other tooltips I've used before automatically adjust the position if it's going to display off screen, but this doesn't.

**Comment by QuentinRoy** [comment]
I agree this is a must have. Tooltips should not end up outside the viewport.

**Comment by peksipatongeis** [comment]
+1 for this

**Comment by musha68k** [comment]
I might need this too, so who of you is implementing it? Dottodot? ;P

**Comment by robclancy** [comment]
I expected this to be a standard feature and part of the point in using a library instead of rolling your own :/
